server.js

```diff
@@ -1729,7 +1729,7 @@ if (config.startServer) {
             (time) => {
               const msg = `BLOCKED: Blocked for ${time}ms (set config.blockedAtWarnEnable for stack trace)`;
               logger.verbose(msg, { time });
-              console.log(msg); // eslint-disable-line no-console
+              console.log(msg);
```

Comment:
Why was this one unused? I thought we _were_ warning on the use of `console.log()`?

Reply:
I can't find any references to that rule ever being enabled in our config 😬 maybe we do want to leave it enabled?

Reply:
Huh! Maybe I'm thinking about PT where do have `'no-console': 'error'` set. I having this set to help catch leftover debugging logs.